dijit/1.11/layout.d.ts

```diff
@@ -318,8 +318,6 @@ declare namespace dijit {
 			 */
 			cancel(): void;
 
-			destroy(): void;
```

Comment:
This already gets inherited and, due to different signature from  `destroy(preserveDom?: boolean): void;`, it causes problems when you try to extend Container and other widgets/mixins.